forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -3808,7 +3808,7 @@ public void commitCrime() {
 
     }
 
-    public int getCommitedCrimeThisTurn() {
+    public int getCommittedCrimeThisTurn() {
         return commitedCrimeThisTurn;
     }
     public void setCommitedCrimeThisTurn(int v) {
```

Comment:
As the one to point it out on Discord, I'd be remiss if I didn't note that lines 3812 and 3814 also have the missing 't' on "committed" typo.